src/presence_analyzer/static/js/mean_time.js

```diff
@@ -8,23 +8,29 @@ function parseInterval(value) {
     $(document).ready(function(){
         var $loading = $('#loading');
 
-        $.getJSON("/api/v1/users", function(result) {
-            var $dropdown = $("#user_id");
+        $.getJSON('/api/v1/users_xml', function(result) {
+            var $dropdown = $('#user_id');
             $.each(result, function(item) {
-                $dropdown.append($("<option />").val(this.user_id).text(this.name));
+                $dropdown.append($('<option />').val(this.user_id).text(this.name));
```

Comment:
It's ok, but this is better:
    
    $dropdown.append(
        $('<option />', {
            'val': this.user_id,
            'text': this.name
        })
    );